pkgs/development/python-modules/diofant/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchPypi
+, glibcLocales, pytestrunner, setuptools_scm
+, mpmath, isort, strategies
+}:
+
+buildPythonPackage rec {
+  pname = "diofant";
+  version = "0.10.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "Diofant";
+    sha256 = "0qjg0mmz2cqxryr610mppx3virf1gslzrsk24304502588z53v8w";
+  };
+
+  checkInputs = [ glibcLocales pytestrunner ];
```

Comment:
Not really important since the tests are disabled, but why do we need `glibcLocales` here?

Reply:
It doesn't compile without this.